magentoerpconnect/sale.py

```diff
@@ -521,6 +534,18 @@ def _clean_magento_items(self, resource):
 
         # Group the childs with their parent
         for item in resource['items']:
+            if phpserialize is None:
+                _logger.info(PHPSERIALIZE_INFO)
```

Comment:
This logger will be logged at each import, I think it would be better to log it once in the `ImportError`.